### ISSUE TITLE
CT Fatal error: There is an issue with the licenses for media types and There is no top chocolate on plate since active chocolates is: 0

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/mean-share-and-balance#108]
```
mean-share-and-balance : build
Build failed with status code 1:
Running "report-media" task
>> missing-file: mean-share-and-balance/images/notebookPaperBackground.png
>> missing-file: mean-share-and-balance/images/labDesk.png
Fatal error: There is an issue with the licenses for media types.
Snapshot from 9/15/2022, 7:07:16 AM

----------------------------------

mean-share-and-balance : interactive-description-fuzz-fuzzBoard-combo : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22interactive-description-fuzz-fuzzBoard-combo%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1663247236549%22%2C%22timestamp%22%3A1663254212089%7D&brand=phet&ea&supportsInteractiveDescription=true&fuzz&fuzzBoard&memoryLimit=1000
Query: brand=phet&ea&supportsInteractiveDescription=true&fuzz&fuzzBoard&memoryLimit=1000
Uncaught Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/assert/js/assert.js:28:13)
at assert (LevelingOutModel.ts:190:14)
at getTopActiveChocolateOnPlate (LevelingOutModel.ts:102:19)
at updateChocolateAmount (LevelingOutModel.ts:136:8)
at listener (TinyEmitter.ts:95:8)
at emit (ReadOnlyProperty.ts:304:22)
at _notifyListeners (ReadOnlyProperty.ts:255:13)
at unguardedSet (ReadOnlyProperty.ts:239:11)
at set (Property.ts:65:10)
at set (Property.ts:58:9)
id: Bayes Puppeteer
Snapshot from 9/15/2022, 7:07:16 AM

----------------------------------

mean-share-and-balance : interactive-description-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22interactive-description-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1663247236549%22%2C%22timestamp%22%3A1663260190684%7D&brand=phet&ea&fuzz&supportsInteractiveDescription=true&memoryLimit=1000
Query: brand=phet&ea&fuzz&supportsInteractiveDescription=true&memoryLimit=1000
Uncaught Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/assert/js/assert.js:28:13)
at assert (LevelingOutModel.ts:190:14)
at getTopActiveChocolateOnPlate (LevelingOutModel.ts:102:19)
at updateChocolateAmount (LevelingOutModel.ts:136:8)
at listener (TinyEmitter.ts:95:8)
at emit (ReadOnlyProperty.ts:304:22)
at _notifyListeners (ReadOnlyProperty.ts:255:13)
at unguardedSet (ReadOnlyProperty.ts:239:11)
at set (Property.ts:65:10)
at set (Property.ts:58:9)
id: Bayes Puppeteer
Snapshot from 9/15/2022, 7:07:16 AM

----------------------------------

mean-share-and-balance : interactive-description-fuzzBoard : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22interactive-description-fuzzBoard%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1663247236549%22%2C%22timestamp%22%3A1663260096548%7D&brand=phet&ea&fuzzBoard&supportsInteractiveDescription=true&memoryLimit=1000
Query: brand=phet&ea&fuzzBoard&supportsInteractiveDescription=true&memoryLimit=1000
Uncaught Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/assert/js/assert.js:28:13)
at assert (LevelingOutModel.ts:190:14)
at getTopActiveChocolateOnPlate (LevelingOutModel.ts:102:19)
at updateChocolateAmount (LevelingOutModel.ts:136:8)
at listener (TinyEmitter.ts:95:8)
at emit (ReadOnlyProperty.ts:304:22)
at _notifyListeners (ReadOnlyProperty.ts:255:13)
at unguardedSet (ReadOnlyProperty.ts:239:11)
at set (Property.ts:65:10)
at set (Property.ts:58:9)
id: Bayes Puppeteer
Snapshot from 9/15/2022, 7:07:16 AM

----------------------------------

mean-share-and-balance : multitouch-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1663247236549%22%2C%22timestamp%22%3A1663253915531%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Uncaught Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/assert/js/assert.js:28:13)
at assert (LevelingOutModel.ts:190:14)
at getTopActiveChocolateOnPlate (LevelingOutModel.ts:102:19)
at updateChocolateAmount (LevelingOutModel.ts:136:8)
at listener (TinyEmitter.ts:95:8)
at emit (ReadOnlyProperty.ts:304:22)
at _notifyListeners (ReadOnlyProperty.ts:255:13)
at unguardedSet (ReadOnlyProperty.ts:239:11)
at set (Property.ts:65:10)
at set (Property.ts:58:9)
id: Bayes Puppeteer
Snapshot from 9/15/2022, 7:07:16 AM

----------------------------------

mean-share-and-balance : multitouch-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1663247236549%22%2C%22timestamp%22%3A1663260969804%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Uncaught Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
Error: Assertion failed: There is no top chocolate on plate since active chocolates is: 0
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1663247236549/assert/js/assert.js:28:13)
at assert (LevelingOutModel.ts:190:14)
at getTopActiveChocolateOnPlate (LevelingOutModel.ts:102:19)
at updateChocolateAmount (LevelingOutModel.ts:136:8)
at listener (TinyEmitter.ts:95:8)
at emit (ReadOnlyProperty.ts:304:22)
at _notifyListeners (ReadOnlyProperty.ts:255:13)
at unguardedSet (ReadOnlyProperty.ts:239:11)
at set (Property.ts:65:10)
at set (NumberPicker.ts:425:24)
id: Bayes Puppeteer
Snapshot from 9/15/2022, 7:07:16 AM
```

**Comment by marlitas** [comment]
Pushed a couple of commits addressing this, and I found the source of the error. The assertion error happens because we were not adjusting chocolates on a plate when removing a person that has more or less chocolates than their declared number.

The above commit seems to work well when someone has too many chocolates and is removed. However, when a person has too few chocolates, for some reason there are more disappearing from the play area than there should be.

Ran out of time for today, and will come back to this tomorrow.
